lightning/Cargo.toml

```diff
@@ -32,7 +32,10 @@ _bench_unstable = []
 no-std = ["hashbrown", "bitcoin/no-std", "core2/alloc"]
 std = ["bitcoin/std"]
 
-default = ["std"]
+# Generates low-r bitcoin signatures, which saves 1 byte in 50% of the cases
+grind_signatures = []
```

Comment:
Maybe lets flip the default? The grinding is quite fast, so in almost no cases should anyone ever care about it. Lets cheat, though, and have `_bench_unstable` rely on the non-grinding :)